lib/OpenQA/Utils.pm

```diff
@@ -399,5 +400,19 @@ sub check_download_whitelist {
     return ();
 }
 
+sub notify_workers {
+    my $ipc = OpenQA::IPC->ipc;
```

Comment:
As I mentioned in #883 , this creates a circular dependency, which seems bad: now `Utils` uses `IPC`, and `IPC` uses `Utils`...